packages/matchbox/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box } from '../Box';
+import { getChild } from '../../helpers/children';
+import { pick } from '../../helpers/props';
+import styled from 'styled-components';
+import { margin, MarginProps } from 'styled-system';
+import Link from './Link';
+
+const Wrapper = styled(Box)`
+  ${margin}
+`;
+
+export type BreadcrumbProps = MarginProps & {
+  children?: React.ReactNode;
+};
+
+const Breadcrumb = React.forwardRef<HTMLDivElement, BreadcrumbProps>(function Breadcrumb(
+  props: BreadcrumbProps,
+  userRef,
+) {
+  const { children, ...rest } = props;
+  const systemProps = pick(rest, margin.propNames);
+  const links = getChild('Breadcrumb.Link', children);
```

Comment:
We might need to use `React.Children.toArray` here instead of getting by `displayName`

I feel like we might have to do this:
```
<Breadcrumb>
  <Link to="/one">
      <Breadcrumb.Link>Link One</Breadcrumb.Link>
  </Link>
  <Link to="/two">
      <Breadcrumb.Link  active>
        Link Two
      </Breadcrumb.Link>
  <Link>
</Breadcrumb>
```